commands/user.go

```diff
@@ -372,6 +399,51 @@ func updateUserEmailCmdF(c client.Client, cmd *cobra.Command, args []string) err
 	return nil
 }
 
+func changePasswordUserCmdF(c client.Client, cmd *cobra.Command, args []string) error {
+	password, _ := cmd.Flags().GetString("password")
+	current, _ := cmd.Flags().GetString("current")
+	hashed, _ := cmd.Flags().GetBool("hashed")
+
+	if password == "" {
+		var confirm string
+		fmt.Printf("Are you changing your own password? (YES/NO): ")
+		fmt.Scanln(&confirm)
+		if confirm == "YES" {
+			fmt.Printf("Current password: ")
+			var err error
+			current, err = getPasswordFromStdin()
+			if err != nil {
+				return errors.New("couldn't read password: " + err.Error())
+			}
+		}
+
+		fmt.Printf("New password: ")
+		var err error
+		password, err = getPasswordFromStdin()
+		if err != nil {
+			return errors.New("couldn't read password: " + err.Error())
```

Comment:
Just curious if there are plans to use `errors.Wrap()` at any point from `github.com/pkg/errors`? I see that it is imported, so was just wondering why we didn't use it.

Reply:
Yep, it makes total sense to wrap the errors in this case. Cobra doesn't do anything special with the wrapped information, but there is no reason not to do it really

Updated where possible :)